src/utils/genesysCloudUtils.tsx

```diff
@@ -0,0 +1,103 @@
+import { clientConfig } from '../clientConfig';
+const platformClient = require('purecloud-platform-client-v2/dist/node/purecloud-platform-client-v2.js');
+const searchApi = new platformClient.SearchApi();
+const usersApi = new platformClient.UsersApi();
+const analyticsApi = new platformClient.AnalyticsApi();
+const tokensApi = new platformClient.TokensApi();
+const routingApi = new platformClient.RoutingApi();
+const presenceApi = new platformClient.PresenceApi();
+
+let offlinePresenceId = '';
```

Comment:
This can/should be hardcoded to `ccf3c10a-aa2c-4845-8e8d-f59fa48c58e5`. _System_ presence IDs are hardcoded into Genesys Cloud, can never change, and are not unique to orgs or regions. When you hardcode this, make a note similar to this explaining why it's hardcoded. User-defined _Org_ presences are not hardcoded though.